### PR TITLE
Build path expr.

### PR DESCRIPTION
**Pull request by CPWstatic**
https://github.com/vesoft-inc/nebula-storage/pull/193
https://github.com/vesoft-inc/nebula-graph/pull/392

### CODE REVIEW

**Review by nevermore3 — DISMISSED** [review]

**Review by nevermore3 — DISMISSED** [review]

File: src/common/expression/PathBuildExpression.cpp [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+    if (value.isStr()) {
+        vertex = Vertex(value.getStr(), {});
+        return true;
+    }
+    if (value.isVertex()) {
+        vertex = Vertex(value.getVertex());
+        return true;
+    }
+    if (value.isInt()) {
+        // TODO:
```
Comment (laura-ding) [inline src/common/expression/PathBuildExpression.cpp:52]: it's the same with string? why add todo here?

File: src/common/expression/PathBuildExpression.cpp [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+                path.steps.emplace_back();
+            }
+        }
+    }
+    result_ = path;
+    return result_;
+}
+
+bool PathBuildExpression::getVertex(const Value& value, Vertex& vertex) const {
+    if (value.isStr()) {
```
Comment (laura-ding) [inline src/common/expression/PathBuildExpression.cpp:43]: Is string means it must the vertex?

File: src/common/expression/PathBuildExpression.cpp (hunk 1 above)
Comment (CPWstatic) [reply src/common/expression/PathBuildExpression.cpp:52]: Integer could not be handled by `Vertex` for now.

File: src/common/expression/PathBuildExpression.cpp (hunk 2 above)
Comment (CPWstatic) [reply src/common/expression/PathBuildExpression.cpp:43]: Yes

**Review by laura-ding — DISMISSED** [review]

**Review by laura-ding — APPROVED** [review]

**Review by nevermore3 — APPROVED** [review]

**Pull request merged**